### PR TITLE
FIX Multiservice default mode for the Context Broker

### LINKED ISSUE #964: Default multiservice option in the RPM
Currently, using the RPM, the Context Broker starts allways in a non-multiservice mode. Considering the Context Broker can work without tenants even in multiservice mode (using requests without fiware-service headers), the multiservice mode should be promoted to the default mode (with the options in the RPM).

### PR DESCRIPTION
**Pull request by dmoranj**
Fixes #964 

Make the multiservice mode the default one in the RPM configuraiton.

### CODE REVIEW

**Comment by dmoranj** [comment]
Fixes #964

File: etc/config/contextBroker [hunk 1]
```diff
@@ -54,3 +54,4 @@ BROKER_DATABASE_NAME=orion
 
 # Use the following variable if you need extra ops
```
Comment (fgalan) [inline etc/config/contextBroker]: CHANGE_NEXT_RELEASE entry is missing (about that the default mode has been changed in the RPM).

File: etc/config/contextBroker [hunk 2]
```diff
@@ -54,3 +54,4 @@ BROKER_DATABASE_NAME=orion
...
 #BROKER_EXTRA_OPS="-t 0-255"
+BROKER_EXTRA_OPS"-multiservice"
```
Comment (kzangeli) [inline etc/config/contextBroker]: Is a '=' missing here?
Should it be:

    BROKER_EXTRA_OPS="-multiservice"

?

File: test/unittests/testData/ngsi9.discoverContextAvailabilityRequest.entityIdTypeAsField.invalid.xml [hunk 3]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (dmoranj) [inline test/unittests/testData/ngsi9.discoverContextAvailabilityRequest.entityIdTypeAsField.invalid.xml]: This file wasn't meant to be there, its presence here is due to a bug in the git implementation for Mac; I can't get rid of it. Please, one of you, remove it from the PR.

File: etc/config/contextBroker (hunk 1 above)
Comment (dmoranj) [inline etc/config/contextBroker]: Fixed in d0950ec

File: etc/config/contextBroker (hunk 2 above)
Comment (dmoranj) [inline etc/config/contextBroker]: Fixed in d0950ec

File: test/unittests/testData/ngsi9.discoverContextAvailabilityRequest.entityIdTypeAsField.invalid.xml (hunk 3 above)
Comment (fgalan) [inline test/unittests/testData/ngsi9.discoverContextAvailabilityRequest.entityIdTypeAsField.invalid.xml]: Rollbacked in ee056be

**Comment by fgalan** [comment]
LGTM

**Pull request merged**